optaplanner-core/src/test/java/org/optaplanner/core/impl/score/director/drools/testgen/fact/TestGenValueFactTest.java

```diff
@@ -40,7 +38,7 @@ public void initialization() {
 
         StringBuilder sb = new StringBuilder(100);
         fact.printInitialization(sb);
-        assertEquals("    private final TestdataValue testdataValue_321 = new TestdataValue();\n", sb.toString());
+        assertEquals("    private final TestdataValue testdataValue_321 = new TestdataValue();" + System.lineSeparator(), sb.toString());
```

Comment:
In that case I would make this test more relaxed and not care about white spaces around the statement. `trim()` before asserting should do it.

The change from `System.lineSeparator()` to `\n` should be done in a new PR. I'd like to take care of it because I also intend to replace some usage of `StringBuilder.append()` with plain string concatenation (`+`) to lower verbosity and increase readability.